src/handlers/event.ts

```diff
@@ -22,6 +23,8 @@ import { tracer as globalTracer } from '../observability/tracer/tracer'
 import { isFunction } from '@skyleague/axioms'
 import type { Context, SNSEventRecord, KinesisStreamRecord, FirehoseTransformationEventRecord, SQSRecord } from 'aws-lambda'
 
+import { randomUUID } from 'crypto'
```

Comment:
Is this a compliant v4 uuid?

Reply:
found my own answer 🙊  https://nodejs.org/api/crypto.html#cryptorandomuuidoptions